.github/workflows/pythonapp.yml

```diff
@@ -84,3 +84,14 @@ jobs:
       with:
         file: ./coverage.xml
         flags: unittests
+    - name: Generate Documentation   # for some awful reason, this only works with older torch????
+      if: ${{ matrix.python-version == '3.7' && matrix.os == 'ubuntu-latest' }}
```

Comment:
should this be 3.5?

Reply:
no, I got it working across all versions...finally.  :)   And since Python3.5 will be deprecated in January, we don't want to use that.

Per our convo this morning, we'll just have one of the targets do the coverage, so i figured Ubuntu  w/3.7 could be the machine that does coverage. But maybe that makes this  target too long.  I'm not sure what other "matrix" ci/cd systems do for things that should  only be run once